pom.xml

```diff
@@ -116,6 +116,17 @@
   </build>
 
   <repositories>
+    <repository>
```

Comment:
Something weird's happening the OpenMRS repo. If an artefact is missing or poor redirection happens, builds fail. That's why I'm leaving it for last to prevent invalid failures.

Reply:
@ibacher in the end something's fishy when there's redirections to another Maven repository. In particular this happens with Iniz because it is proxied maybe? And it looks like letting the build fetch it from the original repo first solves the issue.

In short this proxying is just not working really, we should never have had to add Mekom's repos here, let alone first.

So while I agree that this is not idea altogether, it seems to finally unblock the build.

Reply:
@ibacher, where you able to disable proxying for the Mekom repo on the OpenMRS repos so I could try again re-arranging the repositories?

Reply:
We aren't proxying the Mekom repo in the OpenMRS repo any more... I'm not sure that we have been for a while. What's weird about this to me is that we've been successfully pulling down, e.g., Iniz with the previous ordering.

Reply:
So when I try to run this PR the error I see is this:

```
Execution validate-configurations of goal org.openmrs.maven.plugins:openmrs-packager-maven-plugin:1.7.0:validate-configurations failed: Plugin org.openmrs.maven.plugins:openmrs-packager-maven-plugin:1.7.0 or one of its dependencies could not be resolved: Could not find artifact org.openmrs.module:initializer-validator:jar:2.5.2 in openmrs-repo (https://openmrs.jfrog.io/artifactory/public)
```

There are two entries in the POM, both labelled `openmrs-repo`, one in repositories and one in pluginRepositories. We are able to find Initializer when included as a dependency (it is a dependency of the `distro` sub-project and this works), but we fail to find it when it is a dependency of a Maven plugin. I'm guessing this is because we have a separate pluginRepositories element, but with no reference to the Mekom repos there, which is why we're getting the unresolved thing. This has nothing to do with repo order or proxying or anything like this. We've just told Maven to only search two repositories for plugins and dependencies, and neither of them have Initializer in them.

Adding the Mekom Nexus references to the pluginRepositories element resolves the issue, at least for me.

Reply:
What's weird here is that when running the build in Gitpod, it really works fine.

It's able to fetch the `initializer-validator 2.5.2` as it should (even with the Mekom repos being under the OpenMRS ones, as well as with using `https://mavenrepo.openmrs.org/releases` instead of `https://openmrs.jfrog.io/artifactory/releases`)

Reply:
I might guess that your Gitpod instance had already separately resolved initializer-validator 2.5.2. If the dependency (especially a released dependency) is in the local Maven cache, I don't think Maven worries too much about where it was resolved from.